examples/simple.rs

```diff
@@ -1,12 +1,29 @@
 extern crate oping;
-use oping::{Ping, PingResult};
+use oping::{Ping, PingResult, PingError};
 
 fn do_stuff() -> PingResult<()> {
     let mut ping = Ping::new();
     try!(ping.set_timeout(5.0));  // timeout of 5.0 seconds
     try!(ping.add_host("localhost"));  // fails here if socket can't be created
     try!(ping.add_host("::1"));  // IPv4 / IPv6 addresses OK
-    try!(ping.add_host("1.2.3.4"));
+    
+    /* Replacing the following 'try' macro with an example of how to use
+     * my PingError::CustomError. I suggest adding a broken IP Address like
+     * 1.300.1.1 to see it in action.
+     *      - Timmonfette1
+     *
+     * try!(ping.add_host("1.2.3.4"));
+     */
+    let ip = "1.2.3.4";
+    match ping.add_host(ip) {
+        Ok(_)  => (),
+        Err(e) =>
+        {
+            let error_message = format!("Failure from {} - {}", ip, e);
```

Comment:
Ah -- was the intent to include more information about the failed ping's parameters in the error?

In that case, I'd be fine with extending the error type directly, and modifying the API implementations to add context -- e.g., `add_host()` could return a `PingError::FailedAddHost { hostname: String, error: String }`.